imap_processing/tests/lo/test_lo_l1b.py

```diff
@@ -503,3 +512,67 @@ def test_set_bad_times():
 
     # Assert
     np.testing.assert_array_equal(l1b_de["badtimes"], expected_bad_times)
+
+
+@pytest.mark.external_kernel
+@pytest.mark.use_test_metakernel("imap_ena_sim_metakernel.template")
+def test_set_direction():
+    # Arrange
+    l1b_de = xr.Dataset(
+        {},
+        coords={
+            "epoch": [
+                7.9794907049e17,
+                7.9794907153e17,
+                7.9794907254e17,
+            ],
+        },
+    )
+    # Need to better verify these values. Latitudes are likely to
+    # be 0, but not sure what values to expect from Longitudes.
+    # The expected values are based on what I'm getting from the
+    # SPICE tools.
+    expected_direction_lat = np.array([0, 0, 0])
+    expected_direction_lon = np.array([91.3, 116.3, 140.5])
+
+    # Act
+    l1b_de = set_pointing_direction(l1b_de)
+
+    # Assert
+    np.testing.assert_allclose(
+        l1b_de["direction_lat"].values,
+        expected_direction_lat,
+        atol=1e-1,
+    )
+    np.testing.assert_allclose(
+        l1b_de["direction_lon"].values,
+        expected_direction_lon,
+        atol=1e-1,
+    )
+
+
+def test_pointing_bins():
+    # Arrange
+    l1b_de = xr.Dataset(
+        {
+            "direction_lat": ("epoch", [0, 0, 0]),
+            "direction_lon": ("epoch", [91.3, 116.3, 140.5]),
```

Comment:
Add some negatives in here too to test you handle -180, 180 and not 0, 360 on accident.